corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js

```diff
@@ -1,34 +1,224 @@
 hqDefine("enterprise/js/enterprise_dashboard", [
     'jquery',
+    'knockout',
+    'underscore',
     'hqwebapp/js/initial_page_data',
     'hqwebapp/js/bootstrap5/alert_user',
     'analytix/js/kissmetrix',
+    'hqwebapp/js/tempus_dominus',
+    'moment',
     'hqwebapp/js/bootstrap5/hq.helpers',
+    'hqwebapp/js/bootstrap5/components.ko',
```

Comment:
Can you add a comment that this is for the select toggle? It can be annoying to track down when these implicit dependencies can be removed.

Reply:
Good call out. I had already forgotten why this needed to be included. However, it does feel pretty brittle to rely on comments for this behavior. I could see a situation where this file was changed to have dependencies beyond the select-toggle that were not added to the comment I just left. It makes me wonder if we should be splitting the components into their own files, so the dependency is more explicit.

Reply:
That's a good idea, depending on individual component files instead of one overarching `components.ko.js` file. I can take a stab at that.

Reply:
Started this in https://github.com/dimagi/commcare-hq/pull/35187